webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -3587,7 +3584,89 @@ var userOverride = {
+                        console.log(response);
+                    },
+                    error: function (error) {
+                        console.error(error);
+                    },
+                });
+            },
+            _this.changeConversationAssignee = function () {
+                window.Applozic.ALApiService.ajax({
+                    type: 'PATCH',
+                    url:
+                        MCK_BASE_URL +
+                        CHANGE_BOT +
+                        '?groupId=' +
+                        encodeURIComponent(CURRENT_GROUP_DATA.tabId) +
+                        '&assignee=' +
+                        encodeURIComponent(
+                            CURRENT_GROUP_DATA.initialBot.userId
+                        ),
+                    global: false,
+                    contentType: 'text/plain',
+                    success: function (data) {
+                        if (
+                            data.status == 'success' &&
+                            appOptions.restartConversationByUser &&
+                            CURRENT_GROUP_DATA.conversationAssignee !=
+                                CURRENT_GROUP_DATA.initialBot.userId
+                        ) {
+                            mckGroupService.removeGroupMemberFromChat({
+                                groupId: CURRENT_GROUP_DATA.tabId,
+                                userId: CURRENT_GROUP_DATA.conversationAssignee,
+                                callback: function (data) {
+                                    if (data.status == 'success') {
+                                        _this.triggerWelcomeEvent();
+                                    } else {
+                                        console.error(
+                                            'Error while removing the conversation assignee.'
+                                        );
+                                    }
+                                },
+                            });
+                           
+                        } else {
+                            appOptions.restartConversationByUser &&
+                                _this.triggerWelcomeEvent();
+                        }
+                    },
+                    error: function (data) {
+                        console.log(data);
+                    },
+                });
+            };
+            _this.restartConversation = function (event) {
+                kmWidgetEvents.eventTracking(
+                    eventMapping.onRestartConversationClick
+                );
+                if (
+                    event.target.id == 'km-restart-conversation' ||
+                    ('km-restart-conversation-text' &&
+                        appOptions.restartConversationByUser)
+                ) {
+                    _this.changeConversationAssignee();
+                } else {
+                    KommunicateUI.showClosedConversationBanner(false);
```

Comment:
three dot dropdown btn is not coming after the conversation is resolved and restarted.

Reply:
@sritichamola can you please confirm this flow?

Reply:
It should come even after the conversation is resolved and reopend.

Reply:
Ratified